autoload/fzf/vim.vim

```diff
@@ -1267,10 +1267,14 @@ function! s:given_range(line1, line2)
 endfunction
 
 function! fzf#vim#commits(...) range
+  call winrestview(b:fzf_winview)
```

Comment:
We should guard this code with `if exists('b:fzf_winview')`.

A user may have a custom version of the command directly calling the autoload function and we don't want it to fail after this change.

```sh
command! MyCommits call fzf#vim#commits({'left': '40%', 'options': '--preview-window up'})
```

Reply:
Pushed. Thanks for reviewing :)